ims/rpc/server/rpc_server.py

```diff
@@ -26,6 +34,10 @@ def execute_command(self, credentials, command, args):
             logger.exception('')
             return {constants.STATUS_CODE_KEY: ex.status_code,
                     constants.MESSAGE_KEY: str(ex)}
+        except Exception as ex:
+            logger.exception('')
+            return {constants.STATUS_CODE_KEY: 500,
+                    constants.MESSAGE_KEY: str(ex)}
```

Comment:
Should we return str(ex) directly here? Some failures may not make much sense to end-user.